packages/plugin-core/src/commands/VaultAddCommand.ts

```diff
@@ -140,23 +140,39 @@ export class VaultAddCommand extends BasicCommand<CommandOpts, CommandOutput> {
 
   async handleRemoteRepo(opts: CommandOpts) {
     const baseDir = DendronWorkspace.wsRoot();
-    await this.git.clone(opts.pathRemote!, opts.path);
-    const { vaults } = GitUtils.getVaultsFromRepo({
-      repoPath: path.join(baseDir, opts.path),
-      wsRoot: DendronWorkspace.wsRoot(),
-      repoUrl: opts.pathRemote!,
-    });
-    if (_.size(vaults) === 1 && opts.name) {
-      vaults[0].name = opts.name;
-    }
-    // add all vaults
-    await _.reduce(
-      vaults,
-      async (resp: any, vault: DVault) => {
-        await resp;
-        return this.addVaultToWorkspace(vault);
+    const vaults = await window.withProgress(
+      {
+        location: ProgressLocation.Notification,
+        title: "Adding remote vault",
+        cancellable: true,
```

Comment:
if we want this to be cancellable, we need to pass in a token and add logic to actually cancel this operation. otherwise, i would just set it to false at this point

Reply:
Ah okay. Completely blanked on that. Will change it to not be cancellable.

Reply:
fixed :+1: